userguide/content/en/docs/Getting started/_index.md

```diff
@@ -255,6 +262,30 @@ gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
 To use your own Custom Search Engine, replace the value in the `gcs_engine_id` with the ID of your own search engine.
 
+## Known issues
+
+### MacOS
+
+#### `too many open files` - File Descriptors for "hugo server"
+
+By default, MacOS permits a very small number of open File Descriptors which can cause the following error:
+
+```
+ERROR 2020/04/14 12:37:16 Error: listen tcp 127.0.0.1:1313: socket: too many open files
```

Comment:
Again would it be more helpful to show the hugo error rather than the Mac one, as the hugo error is what users will see at the top of their stack trace? I don't know if any of the users who've had this problem have dug down far enough to get the socket error.

```
fatal error: pipe failed
```

Reply:
This is what is surfaced when when you run `hugo server` (pre postcss v8)

Reply:
As per above let's add both. So.

..which can cause the following errors:

```
(your error
```

or 

```
the pipe failed error
```